install_ddev.sh

```diff
@@ -40,7 +37,7 @@ ${GREEN}
 # It does the following:
 #  -Install Docker for your OS if you don't have it already
 #  -Install ddev by Drud Technology
-#  -Copy required components to ~/Sites/sprint/
```

Comment:
Above here, we should note that this does not install ddev automatically for Windows users.

Reply:
I'm not even sure we should automatically install it for anybody, thinking...

Reply:
I'm pretty sure the fact this doesn't automatically install will mess up the windows test, so still thinking about that too.